### PR TITLE
[WIP] add payu latam gateway

### PR DESCRIPTION
**Pull request by ccarruitero**

### CODE REVIEW

File: lib/spree_gateway/engine.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@ class Engine < Rails::Engine
         app.config.spree.payment_methods << Spree::Gateway::Maxipago
         app.config.spree.payment_methods << Spree::Gateway::Migs
         app.config.spree.payment_methods << Spree::Gateway::SpreedlyCoreGateway
+        app.config.spree.payment_methods << Spree::Gateway::PayuLatamGateway
```
Comment (tvdeyen) [inline lib/spree_gateway/engine.rb]: The Payu Latam gateway only exists in recent versions of ActiveMerchant. Unfortunately Solidus < 1.3 does not support recent versions of ActiveMerchant.

We should only add this gateway if 

```ruby
SolidusSupport.solidus_version > Gem::Version.new("1.3.0")
```

File: spec/models/gateway/payu_latam_gateway_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe Spree::Gateway::PayuLatamGateway do
+  let(:gateway) { described_class.create!(name: 'PayuLatamGateway') }
+
+  context '.provider_class' do
+    it 'is a PayuLatam gateway' do
+      expect(gateway.provider_class).to eq ::ActiveMerchant::Billing::PayuLatamGateway
```
Comment (tvdeyen) [inline spec/models/gateway/payu_latam_gateway_spec.rb]: We should skip this spec if

```ruby
SolidusSupport.solidus_version < Gem::Version.new("1.3.0")
```

**Comment by ccarruitero** [comment]
@tvdeyen thanks for your thoughts.

Btw, I found an error with [solidus credit card](https://github.com/solidusio/solidus/issues/2149) when testing this gateway. If you can take a look to that issue will be useful in order to don't add broken gateways.

**Comment by ccarruitero** [comment]
I found that the gateway require some customization in solidus in order to get work.

So, I moved the code to another [gem](https://github.com/ccarruitero/solidus_payu_latam) to not generate conflicts with other integrations here.

**Comment by tvdeyen** [comment]
Thanks.

**Pull request closed without merging**